scripts/release_scripts/get_repo_specific_changes.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that provides changes specific to oppia repo to be written
+to release summary file.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+from scripts import common
+
+GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING = 'git diff --name-only %s %s'
+GIT_CMD_SHOW_FORMAT_STRING = 'git show %s:feconf.py'
+VERSION_RE_FORMAT_STRING = r'%s\s*=\s*(\d+|\.)+'
+FECONF_VAR_NAMES = ['CURRENT_STATE_SCHEMA_VERSION',
+                    'CURRENT_COLLECTION_SCHEMA_VERSION']
+FECONF_FILEPATH = os.path.join('', 'feconf.py')
+
+
+def get_changes_in_versions(current_release):
+    """Checks if the versions for the exploration or collection schemas have
+    changed.
+
+    Args:
+        current_release: str. The current release tag to diff against.
+
+    Returns:
+        list(str). List of variable names that changed.
+    """
+    feconf_changed_version = []
+    git_show_cmd = (GIT_CMD_SHOW_FORMAT_STRING % current_release)
+    old_feconf = common.run_cmd(git_show_cmd.split(' '))
+    with python_utils.open_file(FECONF_FILEPATH, 'r') as feconf_file:
+        new_feconf = feconf_file.read()
+    for variable in FECONF_VAR_NAMES:
+        old_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, old_feconf)[0]
+        new_version = re.findall(
+            VERSION_RE_FORMAT_STRING % variable, new_feconf)[0]
+        if old_version != new_version:
+            feconf_changed_version.append(variable)
+    return feconf_changed_version
+
+
+def _git_diff_names_only(left, right='HEAD'):
+    """Get names of changed files from git.
+
+    Args:
+        left: str. Lefthand timepoint.
+        right: str. Rightand timepoint.
+
+    Returns:
+        list(str). List of files that are different between the two points.
+    """
+    diff_cmd = (GIT_CMD_DIFF_NAMES_ONLY_FORMAT_STRING % (left, right))
+    return common.run_cmd(diff_cmd.split(' ')).splitlines()
+
+
+def get_changes_in_setup_scripts(base_release_tag, changed_only=True):
+    """Check if setup scripts have changed.
```

Comment:
This is more than just a check, it returns something. Describe clearly here what is being returned (conceptually).

Reply:
Done.